tests/integration/kubernetes/gha-run.sh

```diff
@@ -98,6 +98,20 @@ EOF
 
 	sleep 60s
 	sudo cat "${containerd_config_file}"
+
+	sudo systemctl restart k3s
+
+	local ctr_dm_status
+	local result
+
+	ctr_dm_status=$(sudo ctr plugins ls | awk '$2 ~ /^devmapper$/ { print $0 }' || true)
```

Comment:
I think an argument `--address` should be configured based on the `KUBERNETES` type. For `k3s`, it should be `--address /run/k3s/containerd/containerd.sock`. This is related with a CI failures for s390x:

```
$ sudo ctr --address /run/k3s/containerd/containerd.sock plugins ls | awk '$2 ~ /^devmapper$/ { print $0 }'
io.containerd.snapshotter.v1           devmapper                linux/s390x    ok
$ sudo ctr  plugins ls | awk '$2 ~ /^devmapper$/ { print $0 }'
io.containerd.snapshotter.v1           devmapper                linux/s390x    error
```

This explains that the failure was false-negative.

Reply:
@BbolroC - Thanks for investigating - branch updated.